### PR TITLE
added endpoints, handlers, and accessors

### PR DESCRIPTION
**Pull request by c-Monster**
Naming fixes

### CODE REVIEW

**Comment by xuther** [comment]
Need to merge master back into this branch.

**Comment by jessemillar** [comment]
Set to automatically merge upon successful review. GitHub will handle the magic if you like the code (I still have to test it with the AV API).

File: server.go [hunk 1]
```diff
@@ -42,18 +42,30 @@ func main() {
 	secure.GET("/buildings/:building/rooms/:room/devices", handlerGroup.GetDevicesByBuildingAndRoom)
 	secure.GET("/buildings/:building/rooms/:room/devices/roles/:role", handlerGroup.GetDevicesByBuildingAndRoomAndRole)
 	secure.GET("/buildings/:building/rooms/:room/devices/:device", handlerGroup.GetDeviceByBuildingAndRoomAndName)
-
+	secure.GET("/buildings/:building/rooms/:room/configuration", handlerGroup.GetConfigurationByRoomAndBuilding)
 	secure.PUT("/buildings/:building/rooms/:room/devices/:device/attributes/:attribute/:value", handlerGroup.PutDeviceAttributeByDeviceAndRoomAndBuilding)
 
 	secure.GET("/rooms", handlerGroup.GetAllRooms)
 	secure.GET("/rooms/id/:id", handlerGroup.GetRoomByID)
 	secure.GET("/rooms/buildings/:building", handlerGroup.GetRoomsByBuilding)
```
Comment (xuther) [inline server.go]: This should probably be /buildings/:building/rooms.

Comment (xuther) [inline server.go]: Beyond that one thing the code looks good, once Jesse tests the integration with the av-api, and this gets fixed, we can merge it in.

**Comment by jessemillar** [comment]
We're holding off on approving/merging this until after Monday's demo.

**Pull request closed without merging**